src/main/java/thePackmaster/cards/graveyardpack/SpontaneousRitual.java

```diff
@@ -0,0 +1,132 @@
+package thePackmaster.cards.graveyardpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.GainBlockAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.cards.status.Dazed;
+import com.megacrit.cardcrawl.cards.status.Wound;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.CardStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+
+import basemod.BaseMod;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.actions.graveyardpack.ExhaustFromHandAction;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class SpontaneousRitual extends AbstractGraveyardCard {
+	public static final String ID = makeID("SpontaneousRitual");
+
+	private static final AbstractCard.CardType TYPE = AbstractCard.CardType.ATTACK;
+	private static final AbstractCard.CardRarity RARITY = AbstractCard.CardRarity.UNCOMMON;
+	private static final AbstractCard.CardTarget TARGET = AbstractCard.CardTarget.ENEMY;
+
+	private static final int COST = 1;
+
+	public SpontaneousRitual() {
+		super(ID, COST, TYPE, RARITY, TARGET);
+
+		this.baseDamage = 0;
+		this.damage = this.baseDamage;
+		this.baseMagicNumber=0;
+		this.magicNumber=this.baseMagicNumber;
+		
+	}
+
+	  public void applyPowers() {
+	    super.applyPowers();
+	    
+	    this.baseDamage=SpireAnniversary5Mod.combatExhausts;
+	    this.damage=this.baseDamage;
```

Comment:
This means that the damage won't be modified by strength, etc. Are you sure you want that?

Reply:
Applies strength in testing.